src/test/java/htsjdk/tribble/bed/BEDCodecTest.java

```diff
@@ -226,4 +226,15 @@ private void createIndex(File testFile, File idxFile) throws IOException {
     public void testGetTabixFormat() {
         Assert.assertEquals(new BEDCodec().getTabixFormat(), TabixFormat.BED);
     }
+
```

Comment:
Could you add a small block compressed bedfile and a test that shows that it actually can read from it as well?